git_machete/annotation.py

```diff
@@ -0,0 +1,59 @@
+import re
+from typing import Callable, Optional
+
+from git_machete import utils
+
+
+class Qualifiers:
+    rebase: Optional[bool]
+    merge: Optional[bool]
+    push: Optional[bool]
+
+    def __init__(self, annotation: str):
+        self._annotation = annotation
+        self._annotation_without_qualifiers = annotation
+        self._rebase_text = ''
+        self._push_text = ''
+        self.rebase = None
+        self.push = None
+
+        match_pattern: Callable[[str], str] = lambda text: f'.*\\b{text}=no\\b.*'
```

Comment:
Nit: maybe just make nested `def`s out of these lambdas, more readable

Reply:
I don't know, i like the way it is right now